WordPress/src/main/java/org/wordpress/android/ui/posts/PostListEventListener.kt

```diff
@@ -3,7 +3,7 @@ package org.wordpress.android.ui.posts
 import android.arch.lifecycle.Lifecycle
 import android.arch.lifecycle.LifecycleObserver
 import android.arch.lifecycle.OnLifecycleEvent
-import de.greenrobot.event.EventBus
+import org.greenrobot.eventbus.EventBus
```

Comment:
In this class, there are event handlers like this without `@Subscribe`:

```kt 
fun onEventBackgroundThread(event: UploadService.UploadErrorEvent)
```

Should they also have `@Subscribe`? It does not look like these are getting called.

Reply:
great catch! you're right. I'll fix that